File: app/src/main/java/com/coder/ffmpegtest/utils/ToastUtils.java
package com.coder.ffmpegtest.utils;

import android.widget.Toast;

import com.coder.ffmpegtest.BaseApplication;

/**
 * @author: AnJoiner
 * @datetime: 19-12-20
 */
public class ToastUtils {
    public static void show(String msg) {
        Toast.makeText(BaseApplication.getInstance().getApplicationContext(), msg,
                Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/coder/ffmpegtest/utils/ToastUtils.kt
package com.coder.ffmpegtest.utils

import android.widget.Toast
import com.coder.ffmpegtest.BaseApplication

/**
 * @author: AnJoiner
 * @datetime: 19-12-20
 */
object ToastUtils {
    fun show(msg: String?) {
        Toast.makeText(BaseApplication.Companion.instance?.getApplicationContext(), msg,
                Toast.LENGTH_SHORT).show()
    }
}